tests/js/client/shell/shell-numberof-servers-cluster.js

```diff
@@ -0,0 +1,89 @@
+/*jshint globalstrict:false, strict:false, maxlen : 4000 */
+/* global arango, assertFalse, assertEqual, assertNull  */
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief tests for inventory
+///
+/// @file
+///
+/// DISCLAIMER
+///
+/// Copyright 2010-2012 triagens GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is triAGENS GmbH, Cologne, Germany
+///
+/// @author Jan Steemann
+/// @author Copyright 2012, triAGENS GmbH, Cologne, Germany
```

Comment:
We don't care about that?

Reply:
I don't. I think we should do a global replace on all files. AFAIK Frank has a script for this, and I will ask him if he can run it once some of the huge PRs are merged.